### DISCUSSION TITLE
Getting whole image inference done

### DISCUSSION THREAD

**Discussion by Rajesh-ParaxialTech** [Project-MONAI/MONAI#7656] [General]
Hello
May I know how i can perform "whole image inference" (instead of "sliding window inference") based segmentation training using MONAI functions i.e   
     https://docs.monai.io/en/stable/inferers.html   ?

Thanks

Rajesh

**Comment by KumoLiu** [comment]
Hi @Rajesh-ParaxialTech, just using `SimpleInferer`.
https://github.com/Project-MONAI/MONAI/blob/dev/monai/inferers/inferer.py#L356

Thanks.
